src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -19,29 +19,40 @@ import UIKit
 import UserNotifications
 import ExposureNotification
 
-enum OnboardingPageType: Int, CaseIterable {
-	case togetherAgainstCoronaPage = 0
-	case privacyPage = 1
-	case enableLoggingOfContactsPage = 2
-	case howDoesDataExchangeWorkPage = 3
-	case alwaysStayInformedPage = 4
-
-	func next() -> OnboardingPageType? {
-		OnboardingPageType(rawValue: rawValue + 1)
-	}
 
-	func isLast() -> Bool {
-		(self == OnboardingPageType.allCases.last)
-	}
-}
+final class OnboardingInfoViewController: UIViewController {
 
-extension OnboardingInfoViewController: RequiresAppDependencies {
+	@IBOutlet var imageView: UIImageView!
```

Comment:
All of the `@IBOutlet`s should go to the `// MARK: - Internal` section. Or, even better, make them private and put them into the `// MARK: - Private` section.

Reply:
moved @IBAction funcs to private section